File: alternatingSums/alternatingSums.js
function alternatingSums(a) {
    let sumA = 0,
        sumB = 0;
    a.forEach((e, i) => {
        if (i % 2 == 0) {
            sumA += e;
        } else {
            sumB += e;
        }
    });
    return [sumA, sumB]
}

console.log(alternatingSums([50, 60, 60, 45, 70]))

File: Completed/js/alternatingSums/alternatingSums.py
def alternatingSums(a):
    sumA = 0
    sumB = 0
    for i in range(len(a)):
        if(i % 2 == 0):
            sumA += a[i]
        else:
            sumB += a[i]
    return [sumA, sumB]


print(alternatingSums([50, 60, 60, 45, 70]))
